PayPal/src/main/java/com/braintreepayments/api/paypal/vaultedit/PayPalVaultEditCallback.kt

```diff
@@ -3,14 +3,14 @@ package com.braintreepayments.api.paypal.vaultedit
 import com.braintreepayments.api.core.ExperimentalBetaApi
 
 /**
- * Callback for receiving result of [PayPalClient.edit].
+ * Callback for receiving Edit results
  */
 @ExperimentalBetaApi
 fun interface PayPalVaultEditCallback {
```

Comment:
It looks like this is now the callback used for `PayPalClient.createEditAuthRequest` - so let's rename it to `PayPalEditAuthCallback` and update the docstrings accordingly